docs/source/onboarding_resources/computing/index.rst

```diff
@@ -0,0 +1,359 @@
+#. More complex jobs are handled through Jobmon. While currently the Sim Science research team does not use Jobmon frequently, it might be helpful to understand this process with Module 4 within the IHME Learn training `Computational Infrastructure Level 2  <https://ihme.brightspace.com/d2l/home/7090>`_.
+
+If you need help applying any of these trainings to a practical situation, please ask! 
+
+.. _cluster_access_jupyter:
+
+Accessing Juptyer or RStudio from the Cluster
+---------------------------------------------
+
+The other most common task for a Sim Science researcher on the cluster is to 
+start a Juptyer session. Information on how to do this can be found on the Hub 
+page below. You will also need to update your Bash files in order to complete 
+this, which is covered in depth in the section :ref:`Your Bash Files <cluster_access_bash>`. 
+
+.. todo::
+
+  Add central comp link to this!   
+
+Once you have started a session, you will be able to create code, test simulation 
+results, or do quick calculations. Once you have finished coding, you’ll want to 
+follow the same steps as outlined above in the :ref:`Contributing New Documentation <contributing>` 
+section to save the information on GitHub. All of the same Git commands work on 
+the terminal used for cluster access the same way as for GitHub. 
+
+You will need to make sure that you have cloned your repository and are in the 
+appropriate working directory while logged into the cluster. Then you can add, 
+check the status, commit, and push information in a similar way. For many projects, 
+you will create a new GitHub repository that is research team specific to not 
+disturb the engineers coding workflow. 
+
+One difference in uploading to GitHub is that the cluster will require a password 
+to push information. This is **NOT** your GitHub password, but instead is a unique 
+token that you will need to create. `This website <https://techglimpse.com/git-push-github-token-based-passwordless/>`_ has information on creating a token. Many Sim Science users set their token to 
+never expire and save the token where they can reference it later. However, this 
+might compromise security in some cases, so regenerating a token periodically is 
+best practice. 
+
+.. _cluster_access_other:
+
+Other Useful Cluster Tips
+-------------------------
+
+#. If you get tired of typing long commands, one option is to make an alias. An alias is a shortcut command for commonly typed things. More information on how to do was written by the `Cost Effectiveness team <https://hub.ihme.washington.edu/display/CE/Setting+up+cluster+access>`_ 
+#. When your computer falls asleep, it will stop access to the cluster and cut off any jobs that were currently running. This can be problematic if a notebook needs to run overnight. There are a few different options to account for this including: screen, MOSH, or tmux. If you need to use these, ask a teammate. 
```

Comment:
```suggestion
#. When your computer falls asleep, it will stop access to the cluster and cut off any interactive jobs (i.e. :code:`srun` sessions) that were currently running. This can be problematic if a command needs to run overnight. There are a few different options to account for this including: screen, MOSH, or tmux. If you need to use these, ask a teammate. 
```

This shouldn't kill non-interactive jobs; if you use the recommended scripts to start Jupyter or RStudio, they won't be interactive jobs, so they won't be killed if you get disconnected.